src/main/scala/codacy/detekt/Detekt.scala

```diff
@@ -135,9 +136,10 @@ object Detekt extends Tool {
   private def getRules: Map[DetektCategory, Seq[Rule]] = {
     val config = readEmptyConfigFile
 
-    _root_.codacy.helpers.ResourceHelper
+    codacy.helpers.ResourceHelper
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/app/Codacy/codacy-detekt/pullRequest?prid=4549011)